mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -19,7 +19,7 @@ CLASS net/minecraft/class_2960 net/minecraft/util/Identifier
 		ARG 0 id
 		ARG 1 delimiter
 	METHOD method_12831 isCharValid (C)Z
-		ARG 0 c
+		ARG 0 ch
```

Comment:
Most other places use `c`.

Reply:
The standard shortname for a `char` variable is `ch`.

Reply:
No, at least not in yarn. Most other places use `c`.

Reply:
> 
> 
> No, at least not in yarn. Most other places use `c`.

At least this is the case in `Identifier` class.